src/main/java/svc/data/citations/CitationDataSource.java

```diff
@@ -0,0 +1,12 @@
+package svc.data.citations;
+
+import svc.models.Citation;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public interface CitationDataSource {
```

Comment:
Hmmm...maybe. But I'm not going to do extra work until it's actually deemed necessary. the current features of the app don't require it.